### PR TITLE
[BACKLOG-18498] - Create an easier way for producer step to connect t…

### PR DESCRIPTION
**Pull request by pavel-sakun**
…o a Kafka server

### CODE REVIEW

File: kettle-plugins/kafka/pom.xml [hunk 1]
```diff
@@ -109,7 +109,7 @@
           <instructions>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
             <Export-Package>org.pentaho.big.data.kettle.plugins.kafka.*;version=${project.version}</Export-Package>
-            <Import-Package>org.eclipse.swt*;resolution:=optional,org.pentaho.di.ui.xul*;resolution:=optional,org.pentaho.ui.xul*;resolution:=optional,org.pentaho.di.osgi,org.pentaho.di.core.plugins,*</Import-Package>
```
Comment (DFieldFL) [inline kettle-plugins/kafka/pom.xml]: I think the resolution optional for xul was for the step to work in AEL along with the swt resolution set to optional. @kurt is that correct?

Comment (pavel-sakun) [reply kettle-plugins/kafka/pom.xml]: @DFieldFL shouldn't it be in pentaho-ee instead then?

Comment (DFieldFL) [reply kettle-plugins/kafka/pom.xml]: Both this plugin and the mapped AEL/Spark operation are loaded in AEL. So if this plugin is loaded we need to make sure we aren't requiring any UI libs.

Comment (DFieldFL) [reply kettle-plugins/kafka/pom.xml]: Wait for @kurt to give the final 👍 or 👎 on this one

Comment (pavel-sakun) [reply kettle-plugins/kafka/pom.xml]: @DFieldFL there was warning from karaf-bundle-plugin on unused xul imports, so I cleaned it up.

**Pull request merged**